tracking_manager/models/models.py

```diff
@@ -21,6 +22,43 @@ class Base(models.AbstractModel):
     def is_tracked_by_o2m(self):
         return self._name in self.env["ir.model"]._get_model_tracked_by_o2m()
 
+    def _mail_track(self, tracked_fields, initial_values):
+        changes, tracking_value_ids = super()._mail_track(
+            tracked_fields, initial_values
+        )
+
```

Comment:
Don't add empty line inside a method, as it makes difficult the reading to see the continuity and other syntax highlighting like the comments already act as separators.

Reply:
I've removed the empty lines. I also improve the performance impact of this change a little. When there are fields tracked, than an additional search query is executed to check whether a domain is set on the tracked field. If there is not domain set, the rest of the code is skipped.

Do you still think this one query L36-L42 is to much overhead to keep it in this module?